src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue10987.cs

```diff
@@ -0,0 +1,40 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue10987 : _IssuesUITest
+	{
+		const string Editor = "Editor";
+		const string RTLEditor = "RTLEditor";
+		public Issue10987(TestDevice device) : base(device) { }
+
+		public override string Issue => "Editor HorizontalTextAlignment Does not Works.";
+
+		[Test]
+		[Category(UITestCategories.Editor)]
+		public void EditorPlaceholderRuntimeTextAlignmentChanged()
+		{
+			App.WaitForElement(RTLEditor);
+			App.Tap("Button");
+			VerifyScreenshot();
+		}
+
+		[Test]
+		[Category(UITestCategories.Editor)]
+		public void EditorRuntimeTextAlignmentChanged()
```

Comment:
Also needs to review and update the Android snapshots. 
![image](https://github.com/user-attachments/assets/d6bb6228-31a8-4c74-b40d-9261a93db733)

It is due to: https://github.com/dotnet/maui/pull/27179

Reply:
@jsuarezruiz , I have updated the images. Please let me know if you have any concerns.